media-fonts/source-pro/source-pro-20170111.ebuild

```diff
@@ -28,6 +28,7 @@ RESTRICT="binchecks strip"
 RDEPEND="
 	media-libs/fontconfig
 	cjk? ( media-fonts/source-han-sans )
+	!>=media-libs/freetype-2.12.1
```

Comment:
This needs a revbump. Could you also add it as the first element in RDEPEND? 

I imagine this kind of a blocker could be problematic, since it tells portage that if you install source-pro portage can uninstall the blocker (freetype in this case). Now most likely freetype will have rdeps which then prevents the removal, but I can imagine it'll lead to confusing portage error messages.

Instead I see two other options:

1: What if we didn't add this blocker, but instantly stabilized the new fonts you're adding here, and package.masked source-pro instead marking it to be removed? Then the package.mask message could say to switch using source-code-pro, source-sand and source-serif. 

2: Add the blocker to freetype and revbump freetype (I think you had this at some point?)

Reply:
I prefer 1, because it looks like no one else ran into the bug so far.